infra/mock_api/template/ecs.tf

```diff
@@ -109,21 +106,132 @@ resource "aws_ecs_task_definition" "mock-api-ecs-task-definition" {
         {
           containerPort : 8080
         }
-      ],
+      ]
+      readonlyRootFilesystem = true
+      linuxParameters = {
+        capabilities = {
+          drop = ["ALL"]
+        },
+        initProcessEnabled = true
+      }
+      environment : [
+        {
+          "name" : "POSTGRES_USER"
+          "value" : "${data.aws_ssm_parameter.db_username.value}"
+        },
+        {
+          "name" : "POSTGRES_PASSWORD"
+          "value" : "${data.aws_ssm_parameter.db_pw.value}"
+        },
+        {
+          "name" : "POSTGRES_DB"
+          "value" : "${data.aws_ssm_parameter.db.value}"
+        },
+        {
+          "name" : "DB_HOST"
+          "value" : "test-wic-mt.cfsgsglh2mrn.us-east-1.rds.amazonaws.com"
+        }
+      ]
       logConfiguration = {
         logDriver = "awslogs",
         options = {
-          "awslogs-group"  = "${data.aws_cloudwatch_log_group.mock_api}"
-          "awslogs-region" = "us-east-1"
+          "awslogs-group"         = "mock-api",
+          "awslogs-region"        = "us-east-1",
+          "awslogs-stream-prefix" = "mock-api"
```

Comment:
should there be an environment name somewhere in here? at least for the awslogs-stream-prefix? 🤔

Reply:
right now the way logging groups look something like this:
`awslogs-stream-prefix/container(which contains the env_name)/task_id`

like below &darr;
`csv-handler/test-mock-api-container/ff5f126332ff45b49113cd123d02b155`

The container and task_id came from automatic logging settings from FARGATE. I could try applying a different naming scheme so that the logs are less weird